org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/services/format/DOMDocTypeFormatter.java

```diff
@@ -153,6 +170,9 @@ private void formatDTDNodeDecl(DTDDeclNode nodeDecl, XMLFormattingConstraints pa
 			if (!parameters.isEmpty()) {
 				for (DTDDeclParameter parameter : parameters) {
 					replaceLeftSpacesWithOneSpace(parameter.getStart(), edits);
+					// replace current quote with preferred quote in the case:

```

Comment:
Could add please à sample in comment for the prévois line code with replaceLeftSpacesWithOneSpace...